apps/elasticsearch/README.md

```diff
@@ -0,0 +1,37 @@
+# Elasticsearch tray
+
+## Secure the `elasticsearch` cluster
+
+By default, similarly to the elastic-stack distribution, the `elasticsearch`
+tray is deployed as "unsecured". This is perfectly fine for your test
+environments or an internal usage. But, when you need to expose your cluster or
+activate authentication backends (_e.g._ to access Kibana spaces),
+`elasticsearch` requires that you activate the `xpack.security.enabled` mode.
+
+Arnold's Elasticsearch tray fully automates the security mode configuration if
+you set the `elasticsearch_security_enabled` ansible variable to `true` for
+your customer or environment (extra variables can also be set to customize
+paths and names for SSL certificates):
+
+```yaml
+# group_vars/eugene/staging/main.yml
+
+elasticsearch_security_enabled: true
+
+# Optional:
+elasticsearch_ssl_certificates_path: /path/to/certs
+elasticsearch_ssl_certificates_name: foo.p12
+```
+
+### Generate your CA certificate and private key
+
+```
+$ export ES_RELEASE=7.7.1
+$ export ES_CA_PASSWORD="foo"
+$ docker run --rm -t \
+    docker.elastic.co/elasticsearch/elasticsearch:"${ES_RELEASE}" \
+      bash -c " \
+        elasticsearch-certutil ca -s --pem --out /tmp/ca.zip --pass \"${ES_CA_PASSWORD}\" --ca-dn elasticsearch && \
```

Comment:
`elasticsearch` is the name of the service that will be requested. Should it be configurable?